lib/ImportStatements.js

```diff
@@ -244,6 +244,11 @@ export default class ImportStatements {
       (importStatement: ImportStatement): boolean =>
         !importStatement.isParsedAndUntouched(),
     );
+
+    // appendImport only works when sortImports and groupImports is false
+    if (this.config.get('appendImports')) {
+      result.reverse();
```

Comment:
Does this have the effect of reversing all of the imports? Can you add a test that includes more than one existing import please, to show that the list is appended to and not just reversed?